sessions/css-animations/demo-end/js/index.js

```diff
@@ -0,0 +1,12 @@
+const button = document.querySelector('[data-js="button-position"]');
+
+let positionIndex = 0;
+
+button.addEventListener('click', () => {
+	const newPositionIndex = (positionIndex + 1) % 3;
```

Comment:
Just curious: why?

Reply:
I don't know how to rotate through 3 states in a different way ^^. If you know a better way let me know!